packages/atomic/src/components/search/result-lists/atomic-result-list/atomic-result-list.tsx

```diff
@@ -89,7 +89,7 @@ export class AtomicResultList implements InitializableComponent {
   @BindStateToController('tabManager')
   @State()
   public tabManagerState!: TabManagerState;
-  @State() private resultTemplateRegistered = true;
+  @State() private resultTemplateRegistered = false;
```

Comment:
bugfix: the template should be marked as registered later.
See other -list components